[question] Module '"aedes"' has no exported member 'Aedes'

Hi there, I'm attempting to run the provided [examples](https://github.com/moscajs/aedes/blob/main/docs/Examples.md) (literally copy-paste) but having no success. I have installed version `v1.0.0` with `npm i aedes`  only to find this very basic error upon building:

```cmd
error TS2305: Module '"aedes"' has no exported member 'Aedes'.
```

I have rolled back to version `v0.51.3` and changed the basic example to use the old default import, and it worked. Node is `v22.14`. Is it something else I'm missing? sorry if the question is idiot, but I ran out of ideas and definitely checked if the package was installed.

--- a/package.json
+++ b/package.json
@@ -2,7 +2,12 @@
 	"name": "aedes",
 	"version": "1.0.0",
 	"description": "Stream-based MQTT broker",
-	"exports": "./aedes.js",
+	"exports": {
+		".": {
+			"types": "./aedes.d.ts",
+			"default": "./aedes.js"
+		}
+	},
 	"type": "module",
 	"types": "aedes.d.ts",
 	"scripts": {

